2_base_and_sensor.ipynb

```diff
@@ -573,6 +1073,19 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>What's next</b>\n",
+    "    <p>By comparing the laser scan variable to the room shape, it is possible to \"localize\" the robot absolute position in the room.</p>\n",
```

Comment:
"    <p>By comparing the laser scan variable to the room shape, it is possible to \"localize\" the robot's absolute position in the room.</p>\n",